linux/keyman-config/locale/km_KH.po

```diff
@@ -45,12 +45,12 @@ msgstr "អ្នក​មិន​ត្រូវ​បាន​អនុញ្
 #: keyman_config/install_kmp.py:179
 #, python-brace-format
 msgid "install_kmp.py: error: No kmp.json or kmp.inf found in {package}"
-msgstr ""
+msgstr "install_kmp.py: error: No kmp.json or kmp.inf found in {package}"
 
 #: keyman_config/install_kmp.py:246
 #, python-brace-format
 msgid "install_kmp.py: error: No kmp.json or kmp.inf found in {packageFile}"
-msgstr ""
+msgstr "install_kmp.py: error: No kmp.json or kmp.inf found in {packageFile}"
```

Comment:
While these two lines seem to have been missed, the rest of the changes in the file have proper Khmer.

Reply:
From the crowdin conversation @mcdurdin said
> I think it could be left in English, as it is a technical error.